pkg/cli/config/config.go

```diff
@@ -96,6 +98,52 @@ func GetBackplaneConfiguration() (bpConfig BackplaneConfiguration, err error) {
 	return bpConfig, nil
 }
 
+func getProxyURL(bpConfig BackplaneConfiguration, proxyURLConfig any) string {
+	// as part of OSD-19785, proxy-url in config file is converted to []string from string to allow multiple proxies.
+	// For backward compatibility, get proxy-url as 'any' and check if it is still string (set in past) or array
+	// if string, return it else if array, return the first working proxy
+	switch t := proxyURLConfig.(type) {
+	case []interface{}:
+		logger.Infof("proxy-url array found in config file: %s ", t)
+		return getFirstWorkingProxyURL(bpConfig, proxyURLConfig.([]interface{}))
+	case interface{}:
+		logger.Warn("Change proxy-url in config file to an array. Multiple proxy-urls are now supported. First working proxy-url from the array will be used")
```

Comment:
Are we trying to force the support for multiple proxy URLs for a user, this should ideally not be logged.

Reply:
Fixed. As discussed on slack https://redhat-internal.slack.com/archives/C016S65RNG5/p1706887040612449